enhancements/etcd/disaster-recovery-with-ceo.md

```diff
@@ -85,8 +85,38 @@ This procedure enables to recover from a situation where the control plane certi
 
 ## Proposal
 
+### Backup
+
+1. On all masters create a new backup revision `/etc/kubernetes/static-pod-manifests/backup-N`.
+2. On all masters write `/etc/kubernetes/static-pod-manifests/backup-N/backup.env` file containing 3 environmental
+   variables `CREATED`, `ETCD_REVISION` and `APISERVER_REVISION`.
```

Comment:
this needs to be extended now as we are managing more static pods resources. I think we should also include OCP version.

Reply:
Agree.